Updates to the upgrade procedure and documentation for the new PostGIS container tag (#1705)
CrunchyData/postgres-operator: docs/content/pgo-client/common-tasks.md

#### Create a PostgreSQL Cluster with PostGIS

To create a PostgreSQL cluster that uses the geospatial extension PostGIS, you
can execute the following command, updated with your desired image tag. In the

> example below, the cluster will use PostgreSQL 11.8 and PostGIS 2.5:

Reviewer: ```suggestion
example below, the cluster will use PostgreSQL {{< param postgresVersion >}}  and PostGIS {{< param postgisVersion >}}:
```

Author: Updated, thanks.

```shell

> pgo create cluster hagiscluster --ccp-image=crunchy-postgres-gis-ha --ccp-image-tag=centos7-11.8-2.5-4.4.0

Reviewer: ```suggestion
pgo create cluster hagiscluster --ccp-image=crunchy-postgres-gis-ha --ccp-image-tag={{< param centosBase >}}-{{< param postgresVersion >}}-3.0-{{< param operatorVersion >}}
```

Author: updated.

> pgo create cluster hagiscluster --ccp-image={{< param centosBase >}}-{{< param postgresVersion >}}-{{< param postgisVersion >}}-{{< param operatorVersion >}}

Reviewer: Hmm, is this command correct? It seems to have removed the image name from it, and just has the tag.

```suggestion
pgo create cluster hagiscluster \
  --ccp-image=crunchy-postgres-gis-ha \
  --ccp-image-tag={{< param centosBase >}}-{{< param postgresVersion >}}-{{< param postgisVersion >}}-{{< param operatorVersion >}}
```

Author: Yes, not sure what happened there. Updated, thanks.
